src/cnc/models/config/settings.py

```diff
@@ -184,6 +188,8 @@ def existing_instance_name(self):
 class CORSSettings(BaseModel):
     allowed_origins: List[str]
 
+class ServerlessCDNConfig(BaseModel):
```

Comment:
Is this used anywhere?

Reply:
Modifications Reference: https://github.com/coherenceplatform/cnc/pull/90/commits/7e62798906dcb1450236260512a3de5636f93152